requirements.txt

```diff
@@ -17,3 +17,4 @@ keras==2.2.4
 tqdm==4.31.1
 python-dateutil==2.8.0
 zstandard==0.11.0
+taskcluster==7.0.1
```

Comment:
Since this is only needed for deployment, we could avoid adding it to the requirements and just install it in .taskcluster.yml directly.